### PR TITLE
applications: serial_lte_modem: Remove space in URC message

### PR DESCRIPTION
**Pull request by junqingzou**
This is an continued effort to improve URC in PR#3584.
Customer does now want any space behind ',' in URC.
Also include two minor fixes in slm_at_host:
."#XSLMUAR: " instead of "#SLMUART:"
.Use no stack in sending URC

JIRA reference: NCSIDB-355

Signed-off-by: Jun Qing Zou <jun.qing.zou@nordicsemi.no>

### CODE REVIEW

File: applications/serial_lte_modem/doc/TCPIP_AT_commands.rst [hunk 1]
```diff
@@ -1251,8 +1251,8 @@ Response syntax
    #XTCPCLI: <handle>,<data_mode>
 
 The ``<handle>`` value is an integer.
-When positive, it indicates that it opened successfully.
-When negative, it indicates that it failed to open.
+When positive,it indicates that it opened successfully.
+When negative,it indicates that it failed to open.
```
Comment (ru-fu) [inline applications/serial_lte_modem/doc/TCPIP_AT_commands.rst:1255]: These spaces should not be removed.

**Review by ru-fu — CHANGES_REQUESTED** [review]

**Comment by ru-fu** [comment]
Didn't we just add these spaces in https://github.com/nrfconnect/sdk-nrf/pull/3584 ?

File: applications/serial_lte_modem/doc/TCPIP_AT_commands.rst (hunk 1 above)
Comment (junqingzou) [reply applications/serial_lte_modem/doc/TCPIP_AT_commands.rst:1255]: Sorry, mistake due to quite a labor task

**Comment by junqingzou** [comment]
> Didn't we just add these spaces in #3584 ?
We have had spaces since the beginning of SLM. In fact in #3584 we even added spaces for some HTTP/MQTT URCs to keep SLM in the same format. But now customer requests to remove ALL spaces between each element in URC

**Review by ru-fu — APPROVED** [review]
Doc changes look fine.

**Review by martiles — APPROVED** [review]

**Review by rlubos — APPROVED** [review]
LGTM

**Pull request merged**